project/Dependencies.scala

```diff
@@ -23,7 +23,9 @@ object Dependencies {
      // For Twitter's util functions
     "Twitter Maven Repo"             at "https://maven.twttr.com/",
     // For legacy Snowplow libs
-    ("Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/").withAllowInsecureProtocol(true)
+    ("Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/").withAllowInsecureProtocol(true),
+    // confluent repo
+    "confluent" at "https://packages.confluent.io/maven/"
```

Comment:
Could you leave a code comment explaining what we pull from the confluent repo?  Otherwise it looks a bit mysterious.

Reply:
I can't run `assembly` or `docker:publishLocal` for `beam` project due to following errors

```scala
Error downloading io.confluent:kafka-avro-serializer:5.3.2
...
Error downloading io.confluent:kafka-schema-registry-client:5.3.2
...
```

What's is weird is that `sbt-dependency-graph` plugin's `dependencyList` doesn't list either of above artifacts for neither `beam` nor `common` project.

I added it to resolve the issue but I'll dig further to see where it comes from. It is also weird that somehow a kafka artifact is used to build beam-enrich. Maybe a plugin? I don't know yet.

Reply:
alright, coursier helped with this one,

```
beam (configurations compile, compile-internal, optional, provided, runtime, runtime-internal, test, test-internal)
└─ io.confluent:kafka-avro-serializer:5.3.2
   └─ org.apache.beam:beam-sdks-java-io-kafka:2.30.0
      └─ org.apache.beam:beam-runners-google-cloud-dataflow-java:2.30.0
```

Now I'll add a comment on why confluent registry is added

Reply:
I added the explanation in the github issue instead of the codebase and put a reference